### PR TITLE
Get workflow from config file.

### PR DESCRIPTION
**Pull request by simoneduca**

### CODE REVIEW

File: app/modules/transcribe/subjects.factory.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ require('./transcribe.module.js')
     .factory('SubjectsFactory', SubjectsFactory);
 
 // @ngInject
-function SubjectsFactory($q, localStorageService, zooAPI, zooAPIProject) {
+function SubjectsFactory($q, localStorageService, zooAPI, zooAPIConfig, zooAPIProject) {
```
Comment (camallen) [inline app/modules/transcribe/subjects.factory.js]: is this new param zooAPIConfig magically set somewhere? If so is order here important or is it like global vars?

**Comment by camallen** [comment]
@simone is this all the cases of this configuration? can we update all the `links.workflows[0]` calls?

File: app/modules/transcribe/subjects.factory.js (hunk 1 above)
Comment (rogerhutchings) [inline app/modules/transcribe/subjects.factory.js]: It's injected via `ngInject`, it refers to [zooapi.config.js](https://github.com/zooniverse/shakespeares_world/blob/master/app/modules/zoo-api/zooapi.config.js)

**Comment by camallen** [comment]
@rogerhutchings you happy with the changes?

**Comment by simoneduca** [comment]
@camallen done.

**Comment by rogerhutchings** [comment]
lgtm

**Comment by camallen** [comment]
thanks @simoneduca @rogerhutchings

**Pull request merged**